modules/acs-requirements.adoc

```diff
@@ -25,11 +37,17 @@ For example, Intel processors older than _Sandy Bridge_ and AMD processors older
 To install {product-title}, you must have:
 
 ifdef::openshift[]
-* {ocp} version {ocp-supported-version} or later
+* {ocp} version {ocp-supported-version} or later. For more details about supported self-managed and managed {ocp} versions, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```

Comment:
General comment for the PR. Please replace "For more details" with "For more information".